### PR TITLE
feat(ui/col): add a direction API for col

### PR DESCRIPTION
**Pull request by coder-fang**
## Checklist

---

- [ ] Fix linting errors
- [x] Tests have been added / updated (or snapshots)

## Change information

---

### CODE REVIEW

**Comment by BeADre** [comment]
The `direction` prop should be the same as `Space` component, and you should complete docs(en-us, zh-cn) and declaration files(`varlet-ui/types/*`)

File: packages/varlet-ui/src/col/Col.vue [hunk 1]
```diff
@@ -43,7 +44,7 @@ export default defineComponent({
     const span: ComputedRef<number> = computed(() => toNumber(props.span))
     const offset: ComputedRef<number> = computed(() => toNumber(props.offset))
     const { row, bindRow } = useRow()
-
+    const direction: ComputedRef<string> = computed(() => toString(props.direction))
```
Comment (BeADre) [inline packages/varlet-ui/src/col/Col.vue:47]: it is unnecessary

File: packages/varlet-ui/src/col/props.ts [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import type { PropType } from 'vue'
 import { SizeDescriptor } from './provide'
 
+export type SpaceDirection = 'row' | 'column'
```
Comment (BeADre) [inline packages/varlet-ui/src/col/props.ts:4]: should be `ColDirection `

File: packages/varlet-ui/src/row/props.ts [hunk 3]
```diff
@@ -26,4 +26,8 @@ export const props = {
   onClick: {
     type: Function as PropType<(e: Event) => void>,
   },
+  direction: {
+    type: [String],
+    default: 'row',
+  },
```
Comment (BeADre) [inline packages/varlet-ui/src/row/props.ts:32]: same as col

File: packages/varlet-ui/types/col.d.ts [hunk 4]
```diff
@@ -8,6 +8,7 @@ export type SizeDescriptor = {
 export interface ColProps extends BasicAttributes {
   span?: string | number
   offset?: string | number
+  direction?: string
```
Comment (BeADre) [inline packages/varlet-ui/types/col.d.ts:11]: and `row.d.ts`

**Review by BeADre — APPROVED** [review]

**Comment by haoziqaq** [comment]
This pr has some problem, I will fix it after i merging. Please focus on the following commit

**Pull request merged**